infra/cifuzz/cifuzz_test.py

```diff
@@ -126,16 +126,44 @@ def test_valid(self):
               tmp_dir,
               commit_sha='0b95fe1039ed7c38fea1f97078316bfc1030c523'))
       self.assertTrue(os.path.exists(os.path.join(out_path, 'do_stuff_fuzzer')))
-      run_success, bug_found = cifuzz.run_fuzzers(5, tmp_dir)
-    self.assertTrue(run_success)
-    self.assertTrue(bug_found)
+      with unittest.mock.patch.object(fuzz_target.FuzzTarget,
+                                      'is_reproducible',
+                                      side_effect=[True, False]):
+        run_success, bug_found = cifuzz.run_fuzzers(5, tmp_dir, EXAMPLE_PROJECT)
+        build_dir = os.path.join(tmp_dir, 'out', 'oss_fuzz_latest')
+        self.assertTrue(os.path.exists(build_dir))
+        self.assertNotEqual(0, len(os.listdir(build_dir)))
+        self.assertTrue(run_success)
+        self.assertTrue(bug_found)
+
+  def test_old_bug_found(self):
```

Comment:
is this just repeating the test from #3385? if yes, will it go away once you rebase on CL on top of the another?

Reply:
Merged the two of them.